internal/install/_static/Dockerfile.terraform_deployer

```diff
@@ -1,23 +1,24 @@
-FROM hashicorp/terraform:light as terraform
+FROM ubuntu:20.04
+ENV GCLOUD_SDK_VERSION 370.0.0-0
+ENV TERRAFORM_VERSION 1.1.4
 
-FROM python:3-alpine
+RUN apt-get -qq update \
+  && apt-get install -yq curl apt-transport-https ca-certificates gnupg
 
-# required by gcloud SDK
-RUN apk add --no-cache git openssh curl
+RUN echo "deb [signed-by=/usr/share/keyrings/cloud.google.gpg] http://packages.cloud.google.com/apt cloud-sdk main" | tee -a /etc/apt/sources.list.d/google-cloud-sdk.list \
+  && curl https://packages.cloud.google.com/apt/doc/apt-key.gpg | apt-key --keyring /usr/share/keyrings/cloud.google.gpg  add - \
+  && apt-get update -qq \
+  && apt-get install google-cloud-sdk=${GCLOUD_SDK_VERSION} -yq
```

Comment:
@endorama did you try it locally? it takes ages on my setup unless there is something wrong with it.

Reply:
Takes around 1 minute on my machine. :thinking: It may be something related to Docker Machine?

Reply:
Confirmed, the problem with my machine. It took 4 minutes to load it.